app/assets/stylesheets/forms/_buttons.scss

```diff
@@ -6,12 +6,31 @@
 		@include cta-colors($suffix: "-light");
 	}
 
+	&--inverse {
+		@include cta-text-colors($colors-light, $property: background-color);
+		@include cta-background-colors($colors-light, $property: color);
+	}
+
+	&:disabled {
+		opacity: 0.5;
+		cursor: initial;
```

Comment:
i know we're not supporting IE, but doing `cursor: pointer` works just as well and gives you just that much more coverage. 

i tend to just avoid `initial` in general over explicitly setting properties, but that's probably a preference thing.

Reply:
going to change this to `cursor: default` because the point is to remove the pointer